src/cart_items/dto/findAll-cartItem.dto.ts

```diff
@@ -6,3 +6,16 @@ export class FindAllCartItemDto extends PaginationQueryDto {
   @IsOptional()
   user_id: string;
 }
+
+export class FindAllCartItemByAdminDto extends PaginationQueryDto {
```

Comment:
Is it necessary to add the limit and offset in dto? :)

Reply:
Removed